elifetools/utils.py

```diff
@@ -271,7 +272,14 @@ def node_contents_str(tag):
     """
     if tag is None:
         return None
-    return "".join(list(map(unicode_value, tag.children))) or None
+    tag_string = ''
+    for child_tag in tag.children:
+        if isinstance(child_tag, Comment):
+            # BeautifulSoup does not preserve content tags, add them back
```

Comment:
`s/content/comment`?

Reply:
Thanks! I must have `content` on the brain